### PR TITLE
Begin non-special-report landing pages.

### PR DESCRIPTION
**Pull request by kavanagh**
This adds article copy to the PLP and the new layout for the teasers. 

The slate teaser styles are here just to get started with but I'll move them in n-teaser next.

### CODE REVIEW

File: client/components/article/_main.scss [hunk 1]
```diff
@@ -30,6 +30,23 @@
 
 .article__main {
 	padding-top: 15px;
+	padding-bottom: 30px;
+	&-content {
+		@include oGridCenter;
```
Comment (adgad) [inline client/components/article/_main.scss]: Should these be __ ?

File: server/model/package.js [hunk 2]
```diff
@@ -83,13 +100,11 @@ module.exports = function decoratePackage (req, res, payload, flags) {
 					case 'special-report':
```
Comment (adgad) [inline server/model/package.js]: Is this redundant now (there's something similar above)?

Comment (kavanagh) [inline server/model/package.js]: The messages are slightly different depending on the teaser collection.

File: client/components/article/_main.scss (hunk 1 above)
Comment (kavanagh) [inline client/components/article/_main.scss]: I'll change it

**Pull request merged**